backend/packages/Upgrade/src/api/repositories/LogRepository.ts

```diff
@@ -164,7 +164,11 @@ export class LogRepository extends Repository<Log> {
       .innerJoin('queries.metric', 'metric')
       .innerJoin('metric.logs', 'logs')
       .where('experiment.id=:experimentId', { experimentId })
-      .andWhere('logs."userId" IN (:...userIds)', { userIds })
```

Comment:
The where clause here will restrict the list of `userIds` to 50.
Both the codes are doing identical things because after unnesting you are returning 
`'logs."userId" IN ' + subQuery`

Reply:
They were not restricted to 50 when I tested it locally, or when the bug was reported in production. It exceeded the limit of ~64k parameters in postgres. UNNEST fixes it by converting the list of parameters to a single parameter, which is converted by postgres into a native list.